src/components/user-profile.component.js

```diff
@@ -144,7 +154,15 @@ export const UserProfile = ({
       </TouchableOpacity>
 
       {type !== 'org' &&
-        <TouchableOpacity style={styles.unit}>
+        <TouchableOpacity
+          style={styles.unit}
+          onPress={() =>
+            navigation.navigate('StarredRepositoryList', {
+              title: translate('common.stars', language),
```

Comment:
@nersoh please do by analogy as [here](https://github.com/gitpoint/git-point/blob/master/src/locale/languages/en.js#L163), i.e. will be like this:

```js
stars: {
  title: 'Stars', // used on 161 line
  text: 'Stars' // used on 170 line
}
```
This separation is necessary for languages where declination is used. In English there will always be "Stars" in both the `title` and `text`.